fetch.bs

```diff
@@ -4276,84 +4270,93 @@ steps:
 
 <h3 id=cors-preflight-cache>CORS-preflight cache</h3>
 
-<p>A <dfn id=concept-cache>CORS-preflight cache</dfn> consists of a collection of
-entries where each entry has these fields:
+<p>A user agent has an associated <a>CORS-preflight cache</a>. A
+<dfn id=concept-cache>CORS-preflight cache</dfn> is a <a for=/>list</a> of <a>cache entries</a>.
+
+<p>A <dfn>cache entry</dfn> consists of:
 
 <ul class=brief>
- <li><dfn id=concept-cache-origin for=cache>serialized origin</dfn> (a <a for=/>byte sequence</a>)
- <li><dfn id=concept-cache-url for=cache>url</dfn> (a <a for=/>URL</a>)
- <li><dfn id=concept-cache-max-age for=cache>max-age</dfn> (a number of seconds)
- <li><dfn id=concept-cache-credentials for=cache>credentials</dfn> (a boolean)
- <li><dfn id=concept-cache-method for=cache>method</dfn> (null, `<code>*</code>`, or a
+ <li><dfn id=concept-cache-origin for="cache entry">serialized origin</dfn> (a
+ <a for=/>byte sequence</a>)
+ <li><dfn id=concept-cache-url for="cache entry">url</dfn> (a <a for=/>URL</a>)
+ <li><dfn id=concept-cache-max-age for="cache entry">max-age</dfn> (a number of seconds)
+ <li><dfn id=concept-cache-credentials for="cache entry">credentials</dfn> (a boolean)
+ <li><dfn id=concept-cache-method for="cache entry">method</dfn> (null, `<code>*</code>`, or a
  <a for=/>method</a>)
- <li><dfn id=concept-cache-header-name for=cache>header name</dfn> (null, `<code>*</code>`, or a
- <a for=/>header</a> <a for=header>name</a>)
+ <li><dfn id=concept-cache-header-name for="cache entry">header name</dfn> (null, `<code>*</code>`,
+ or a <a for=/>header</a> <a for=header>name</a>)
 </ul>
 
-<p>Entries must be removed after the seconds specified in the
-<a for=cache>max-age</a> field have passed since storing the entry.
-Entries may be removed before that moment arrives.
+<p><a>Cache entries</a> must be removed after the seconds specified in their
+<a for="cache entry">max-age</a> field have passed since storing the entry. <a>Cache entries</a> may
+be removed before that moment arrives.
 
-<p>To <dfn id=concept-cache-create-entry for=cache>create a new entry</dfn> in the
-<a>CORS-preflight cache</a>, given <var>request</var>, <var>max-age</var>,
-<var>method</var>, and <var>headerName</var>, do so as follows:
+<p>To <dfn id=concept-cache-create-entry>create a new cache entry</dfn>, given <var>request</var>,
+<var>max-age</var>, <var>method</var>, and <var>headerName</var>, run these steps:
 
-<dl>
- <dt><a for=cache>serialized origin</a>
- <dd>The result of <a>serializing a request origin</a> with <var>request</var>
+<ol>
+ <li>
+  <p>Let <var>entry</var> be a <a>cache entry</a>, initialized as follows:
 
- <dt><a for=cache>url</a>
- <dd><var>request</var>'s <a for=request>current url</a>
+  <dl>
+   <dt><a for=cache>serialized origin</a>
```

Comment:
In this `<dl>`, `for=cache` should be `for="cache entry"` (or remove the `for` from each item and add `link-for="cache entry"` to the `<dl>`.